lib/dndstrings-captions.sty

```diff
@@ -195,4 +195,64 @@
 %    \renewcommand\spelldurationname{Duration}
   }
 
+% German captions
+\addto\captionsngerman{%
+    \def\armorclassname{Rüstungsklasse}%
+    \def\hitpointsname{Trefferpunkte}%
+    \def\speedname{Bewegungsrate}%
+    \def\strstatname{Str}%
```

Comment:
In the English version of the monster stat blocks, the abbreviated ability scores are all caps (STR, DEX, CON, INT, WIS, CHA). Should the German version of these abbreviations be all caps as well?

Reply:
I agree, just had the character sheet at hand. And there the short version is not all caps. But same is true for the English version.